### PR TITLE
161348758 User should be able to create an article

### PR DESCRIPTION
**Pull request by judeinno**
#### What does this PR do?

Add feature cor create an article.

#### Description of Task to be completed?

- Add user create article form.
- Add drop down for articles to create articles and get all articles
- Add container for create article form.
- Add component for create article form.
- Add actions and reducers for create article.
- Add tests

#### How should this be manually tested?

- Go to create article URL: ````/createArticle````
- Input the article data:
````
- Title
- Description
- Tags
- Image
- Body

````

#### What are the relevant pivotal tracker stories?

[#161348758](https://www.pivotaltracker.com/story/show/161348758)

#### Screenshots 
````
Input UI
````
image

image

````
When Get all articles is active (Dropdown changes fields)
````
image

````
When create article is active(Dropdown changes fields)
````
image

### CODE REVIEW

**Comment by dnuwa** [comment]
kindly fix the merge conflicts

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+APP_URL=https://ah-backend-thanos-staging.herokuapp.com/api
```
Comment (tibetegya) [inline .env]: Add a new line at the end of file

File: src/actions/createArticleActions/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  type: ACTION_TYPE.CREATE_ARTICLE_FAILED,
+  errorMessage,
+});
+
+export const updateImageUrl = imageUrl => ({
+  type: ACTION_TYPE.UPDATE_IMAGE_URL,
+  payload: imageUrl,
+});
+
+const tok = localStorage.getItem('token');
```
Comment (tibetegya) [inline src/actions/createArticleActions/index.js]: What does tok mean??

File: src/containers/CreateArticlePage/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    articleData({
+      ...createArticleReducer.articlePostData,
+      [event.target.name]: event.target.value,
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { postArticle, createArticleReducer } = this.props;
+    postArticle({ ...createArticleReducer.articlePostData, tag_list: createArticleReducer.articlePostData.tag_list.split(', ') });
```
Comment (tibetegya) [inline src/containers/CreateArticlePage/index.js]: this line is too long

**Comment by tibetegya** [comment]
Resolve merge conflicts

**Review by dnuwa — DISMISSED** [review]

File: src/actions/createArticleActions/index.js (hunk 2 above)
Comment (judeinno) [reply src/actions/createArticleActions/index.js]: Am going to update it

File: src/containers/CreateArticlePage/index.js (hunk 3 above)
Comment (judeinno) [reply src/containers/CreateArticlePage/index.js]: Fixed It

**Review by brucemakallan — DISMISSED** [review]
Tried this out on Heroku. All good.

**Review by dnuwa — DISMISSED** [review]

**Review by Sulaiman-Mozes — DISMISSED** [review]

**Review by dnuwa — APPROVED** [review]

**Review by Sulaiman-Mozes — APPROVED** [review]

**Review by RachaelNantale — APPROVED** [review]

**Pull request merged**